### PR TITLE
CNV-1396_2 Fixing Vikram's comments from merged PR 

### PR DESCRIPTION
**Pull request by bgaydosrh**
This PR fixes issues raised by @vikram-redhat in merged PR https://github.com/openshift/openshift-docs/pull/25231

Link to original story: https://issues.redhat.com/browse/CNV-1396

Peer review needed. Label *[enterprise-4](https://issues.redhat.com/browse/enterprise-4).6* only

### CODE REVIEW

**Review by adellape — APPROVED** [review]
This LGTM: ifdef'd items now show up in the rendered assembly.

**Comment by vikram-redhat** [comment]
Needs QE ack to verify that the addition of the blockstorage in the code samples makes sense.

**Comment by bgaydosrh** [comment]
Eliminated conditionals from both module and assembly. @aglitke approved new YAML and suggested accessmode parm be ReadWriteMany not ReadWriteOnce. Revised some text to reflect block mode is used in this example.

File: modules/virt-cloning-a-datavolume.adoc [hunk 1]
```diff
@@ -2,36 +2,28 @@
...
-ifdef::blockstorage[]
-`volumeMode: Block` so that an available block PV is used,
-endif::[]
-and the size of the new DataVolume.
-+
-For example:
+. Create a YAML file for a `DataVolume` object that specifies the name of the
+new DataVolume, the name and namespace of the source PVC, and the size of the new DataVolume. This example clones a source PVC in block mode, so `volumeMode: Block` is used:
 +
+
```
Comment (adellape) [inline modules/virt-cloning-a-datavolume.adoc:26]: White space isn't needed here.

**Comment by aburdenthehand** [comment]
@bgaydosrh I see that this has 'enterprise-4.6 only' and was not cherry-picked to enterprise-4.7. This has created a bunch of merge conflicts trying to cherry-pick #27562 to enterprise-4.7 now and it's difficult to see how the changes in this PR don't also apply to enterprise-4.7. Since this PR was initially created a few days before enterprise-4.7 was created, I'm going to presume that it was an oversight and this PR should have been updated to also be cherry-picked to enterprise-4.7, which I'll do now so that it's in line with master and e-4.6. 
If this is in error, I'll take care of undoing it. 

@vikram-redhat in case you have more context

**Pull request merged**